Sources/MessagingInApp/Gist/Gist.swift

```diff
@@ -135,16 +135,9 @@ public class Gist: GistInstance, GistDelegate {
     }
 
     func logMessageView(message: Message) {
```

Comment:
This function does 2 things: Tracks `opened` metrics and makes sure that a message is not shown again. 

This is the same function used by modal messages to make messages not display twice.